docs/02-get-started/03-deploy-expose-microservice.md

```diff
@@ -0,0 +1,240 @@
+
+<div tabs name="Create a Service" group="deploy-expose-microservice">
+  <details open>
+  <summary label="kubectl">
+  kubectl
+  </summary>
+
+Run:
+
+```bash
+cat <<EOF | kubectl apply -f -
+  apiVersion: v1
+  kind: Service
+  metadata:
+    name: orders-service
+    namespace: default
+    labels:
+      app: orders-service
+      example: orders-service
+  spec:
+    type: ClusterIP
+    ports:
+      - name: http
+        port: 80
+        protocol: TCP
+        targetPort: 8080
+    selector:
+      app: orders-service
+      example: orders-service
+EOF
+```
+
+To check that the Service was created successfully, run: 
+```bash
+kubectl get service orders-service -o=jsonpath="{.metadata.uid}"
+```
+
+The operation was successful if the command returns the **uid** of your Service.
+
+  </details>
+  <details>
+  <summary label="Kyma Dashboard">
+  Kyma Dashboard
+  </summary>
+
+As you've already created the Service in step 3 in the [previous section](#deploy-the-microservice), skip this part.
+
+<!--
+//TODO: Functionality not added yet. Check with Hasselhoffs in a while.
+If you created the Service at the previous step while creating the Deployment, skip this section. Otherwise, you must now create the Service.
+
+1. From the left navigation, go to **Services**.
+2. Click on **Create Service +**.
+3. ...
+
+The operation was successful if ... .
+--->
+  </details>
+</div>
+
+## Expose the microservice
+
+We have the Service created. Let's now expose it outside the cluster.
+
+To expose our microservice, we must create an [APIRule](../05-technical-reference/00-custom-resources/apix-01-apirule.md) CR for it, just like when we [exposed our Function](02-deploy-expose-function.md#expose-the-function).
+
+<div tabs name="Expose the microservice" group="deploy-expose-microservice">
+  <details open>
+  <summary label="kubectl">
+  kubectl
+  </summary>
+
+Run:
+
+```bash
+cat <<EOF | kubectl apply -f -
+apiVersion: gateway.kyma-project.io/v1alpha1
+kind: APIRule
+metadata:
+  name: orders-service
+  namespace: default
+  labels:
+#    app: orders-service
+#    example: orders-service
+spec:
+  service:
+    host: orders-service.$CLUSTER_DOMAIN
+    name: orders-service
+    port: 80
+  gateway: kyma-gateway.kyma-system.svc.cluster.local
+  rules:
+    - path: /.*
+      methods: ["GET","POST"]
+      accessStrategies:
+        - handler: noop
+      mutators: []
+EOF
+```
+
+  </details>
+  <details>
+  <summary label="Kyma Dashboard">
+  Kyma Dashboard
+  </summary>
+
+1. In your Services's view, click on **Expose Service +**.
+2. Provide the **Name** (`hello-world`) and **Hostname** (`hello-world`) and click **Create**.
+
+> **NOTE:** Alternatively, from the left navigation go to **APIRules**, click on **Create apirules +**, and continue with step 2, selecting the appropriate **Service** from the dropdown menu.
+  </details>
+</div>
+
+### Verify the microservice exposure
```

Comment:
I wanted to have it like this to emphasise that it's a subsection. It's just verification. It's okay if it's hidden, because no-one is going to be looking for this specific part, unless they're interested in exposing a Function. 
The main sections of the guide are "deploy" and "expose" and this is supposed to be "in the shadows" :)